src/components/pages/deployments/deploymentDetails/deploymentDetails.js

```diff
@@ -154,59 +162,92 @@ export class DeploymentDetails extends Component {
               <div className="deployment-name">
                 {name}
               </div>
-              <StatSection className="summary-container-row1">
-                <StatGroup className="summary-container-columns">
-                  <StatProperty
-                    value={appliedCount}
-                    label={t('deployments.details.applied')}
-                    size="large" />
-                </StatGroup>
-                <StatGroup className="summary-container-columns">
-                  <StatProperty
-                    value={failedCount}
-                    label={t('deployments.details.failed')}
-                    svg={failedCount !== 0 ? svgs.failed : undefined}
-                    svgClassName="stat-failed"
-                    size="large" />
-                </StatGroup>
-                <StatGroup className="summary-container-columns">
-                  <StatPropertyPair label={t('deployments.details.deviceGroup')} value={deviceGroupName} />
-                </StatGroup>
-                <StatGroup className="summary-container-columns">
-                  <StatPropertyPair
-                    label={t('deployments.details.packageType')}
-                    value={type ? getPackageTypeTranslation(type, t) : undefined} />
-                </StatGroup>
-                <StatGroup className="summary-container-columns">
-                  <StatPropertyPair label={t('deployments.details.priority')} value={priority} />
-                </StatGroup>
-              </StatSection>
-              <StatSection className="summary-container-row2">
-                <StatGroup className="summary-container-columns">
-                  <StatProperty
-                    value={targetedCount}
-                    label={t('deployments.details.targeted')}
-                    size="large" />
-                </StatGroup>
-                <StatGroup className="summary-container-columns">
-                  <StatProperty
-                    className="summary-container-succeeded"
-                    value={succeededCount}
-                    label={t('deployments.details.succeeded')}
-                    size="small" />
-                  <StatProperty
-                    className="summary-container-pending"
-                    value={pendingCount}
-                    label={t('deployments.details.pending')}
-                    size="small" />
-                </StatGroup>
-                <StatGroup className="summary-container-columns">
-                  <StatPropertyPair
-                    label={t('deployments.details.start')}
-                    value={TimeRenderer({ value: createdDateTimeUtc })} />
+              <StatSection>
```

Comment:
How do these wrap if the browser is smaller? When reviewing the mocks vs the current screens for this feature, we talked about reworking this so the wrapping worked differently than it did.  Come chat in person, please.

Reply:
done